extensions/Cake.Liquibase.yml

```diff
@@ -9,3 +9,4 @@ Description: Addin for running liquibase database migrations with the Cake build
 Categories:
 - database
 TargetCakeVersion: 0.36.0
+AnalyzedPackageVersion: 0.1.11-preview
```

Comment:
Here's an example of an addin with no stable release. Therefore AddinDisco analyzed the most recent prerelease package:

![image](https://user-images.githubusercontent.com/112710/101056801-55a26680-3559-11eb-9223-10e66b7adf9c.png)